fitbit-react/src/Pages/Homepage/FeaturedBrands/FeaturedBrands.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "./featuredbrands.css";
+function FeaturedBrands() {
+  return (
+    <div>
+      <h2 class="h3 centered-text home-section-heading">Featured Brands</h2>
+      <div class="spacer-3"></div>
```

Comment:
Suggestion: use `className` instead of  `class` because class is preserved keyword in react

Reply:
Done